### PR TITLE
Shortened names, modularized model, added config.py, added support for tensorboard

### PR DESCRIPTION
**Pull request by mrcolo**

### CODE REVIEW

File: multivariate_single_forecast.py [hunk 1]
```diff
@@ -1,128 +1,119 @@
...
+  # evaluation interval
+  window = int(e.data.shape[0]/BATCH_SIZE) * 1
+  
+  # pick selected features
+  features = e.data[FEATURES]
+  
+  # cleanse
+  features.index = e.data.index
+  features = features.dropna()
+  features = features[26:]
```
Comment (elliotvilhelm) [inline multivariate_single_forecast.py]: I guess this line is not needed, this was meant to drop NA because of moving average, since we use MA_long = 52 then we would need to drop the first 52, but I think `dropna()` above takes care of this. 

Also, 52 MA kind of big, we loose 52 days of data, maybe not worth it.

File: multivariate_single_forecast.py [hunk 2]
```diff
@@ -1,128 +1,119 @@
...
+  ssm = get_lstm()
+
+  # run trial
+  history = ssm.fit(t_ds, epochs=EPOCHS,
+                    steps_per_epoch=window,
+                    validation_data=v_ds,
+                    validation_steps=50, 
+                    callbacks=[v_cb, ts_cb])
+
+  # TODO plot results
```
Comment (elliotvilhelm) [inline multivariate_single_forecast.py]: Lets delete from here down for now, we can add it back if we want it or migrate it to a function.

File: multivariate_single_forecast.py [hunk 3]
```diff
@@ -1,128 +1,119 @@
-from market import EquityData
-from models.lstm import split, split_multivariate, show_plot
+# INSERT COPYRIGHT HERE
```
Comment (elliotvilhelm) [inline multivariate_single_forecast.py]: we can add later, no worry for now

File: multivariate_single_forecast.py [hunk 4]
```diff
@@ -1,128 +1,119 @@
...
-                   'Single Step Prediction')
-  plot.show()
+def get_lstm():
+  """
+  Keras LSTM Architecture 
+  """
+  ssm = tf.keras.models.Sequential()
+
+  ssm.add(tf.keras.layers.LSTM(64, return_sequences=True,
+                                input_shape=xt.shape[-2:]))
```
Comment (elliotvilhelm) [inline multivariate_single_forecast.py]: what is xt?

Comment (mrcolo) [reply multivariate_single_forecast.py]: xtrain

Comment (mrcolo) [reply multivariate_single_forecast.py]: oh okay yea, ill fix that rn

File: multivariate_single_forecast.py (hunk 2 above)
Comment (mrcolo) [reply multivariate_single_forecast.py]: sounds good, so you wanna get rid of tensorboard?

File: multivariate_single_forecast.py (hunk 4 above)
Comment (elliotvilhelm) [reply multivariate_single_forecast.py]: so I guess this is the n_dimensions? based off input dimension, like OHLC would be 4 dimensions?

OHLC + MA = 5 dim? right?

Comment (elliotvilhelm) [reply multivariate_single_forecast.py]: so maybe this should be set to `len(FEATURES)`?

File: multivariate_single_forecast.py (hunk 2 above)
Comment (elliotvilhelm) [reply multivariate_single_forecast.py]: no keep tensorboard, `ts_cb = tf.keras.callbacks.TensorBoard(log_dir=logdir)` handles tensorboard, I was saying delete from line 104 down, just the commented stuff

File: config.py [hunk 5]
```diff
@@ -0,0 +1,10 @@
+BATCH_SIZE = 8
+BUFFER_SIZE = 10000
+EPOCHS = 10
+CLASSIFICATION = True
+STEP = 1
+HISTORY_SIZE = 180
+TARGET_DIS = 6
+FEATURES = ['close', 'MA_long', 'MA_short']
+DATA_DIR = 'data/SPY.csv'
+DATA_SYM = 'SPY'
```
Comment (elliotvilhelm) [inline config.py]: new line at end of file pls

File: multivariate_single_forecast.py (hunk 4 above)
Comment (mrcolo) [reply multivariate_single_forecast.py]: I don't think so, that's the 3d tensor that you input in keras. your old code is doing the same ?

image

File: multivariate_single_forecast.py (hunk 1 above)
Comment (mrcolo) [reply multivariate_single_forecast.py]: ill let that decision be for a later chat

File: multivariate_single_forecast.py (hunk 2 above)
Comment (mrcolo) [reply multivariate_single_forecast.py]: oh 104 down, awesome

File: config.py (hunk 5 above)
Comment (mrcolo) [reply config.py]: apologies

File: multivariate_single_forecast.py (hunk 4 above)
Comment (elliotvilhelm) [reply multivariate_single_forecast.py]: yeah so we have `(batch_size, time_steps, number_of_features)`

for input shape I think we drop batch_size, so `(time_steps, number_of_features)`, so I think its `(HISTORY_SIZE, len(FEATURES))`

can you print what `xt[-2:]` was and confirm?

Comment (elliotvilhelm) [reply multivariate_single_forecast.py]: Id rather have this value be coded and not set dynamically by dataset

Comment (mrcolo) [reply multivariate_single_forecast.py]: I see what ur saying, will work on it later

Comment (mrcolo) [reply multivariate_single_forecast.py]: input makes sense, in the process of changing it to static

Comment (mrcolo) [reply multivariate_single_forecast.py]: got it
